cypress.config.ts

```diff
@@ -15,5 +15,6 @@ export default defineConfig({
     viewportWidth: 1920,
     viewportHeight: 1080,
     experimentalRunAllSpecs: true,
+    video: true
```

Comment:
I don't see the added value of enabling video, we already know that the tests fail occasionally because they're instable. Enabling video will only make the tests slower.

Reply:
If the tests fail, you only have a screenshot where the assertion is not met. Video may be useful to investigate the root cause. We can turn of video again when the tests are stable again.

Reply:
But we already know why the tests fail, because they're unstable, so I don't see the added value of enabling video.